css/index.css

```diff
@@ -0,0 +1,517 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: acumin-pro, "Acumin Pro", "Helvetica Neue", Helvetica, Arial, sans-serif;
```

Comment:
- [ ] Kindly ensure to use `Cocogoose, Lato` fonts in your styling for both pages as they are the fonts used originally in designing these pages.